client/App.jsx

```diff
@@ -7,17 +8,45 @@ class App extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      restaurantInfo: {},
+      showResults: false,
+    };
+  }
+
+
+  componentDidMount() {
+    const restaurantId = window.location.pathname;
+    $.ajax({
+      url: `/api${restaurantId}`,
+      method: 'GET',
+      success: (response) => {
+        console.log(response);
+        this.setState({
+          restaurantInfo: response,
+        });
+      },
+    });
   }
 
   render() {
+    const { restaurantInfo } = this.state;
+    const {
+      name,
+      open,
+      close,
+      min,
+      max,
+      dates,
```

Comment:
most likely you already caught this typo? an extra comma after 'dates'

Reply:
this is one of the Airbnb styleguide recommendations in which you should be leaving trailing commas, even on the last element